docs/manage/backend.md

```diff
@@ -1,90 +1,30 @@
 ---
 myst:
   html_meta:
-    "description": "Manage add-ons, packages, and processes"
-    "property=og:description": "Manage add-ons, packages, and processes"
-    "property=og:title": "Manage add-ons, packages, and processes"
-    "keywords": "Plone 6, manage, backend, add-ons, packages, processes, cookiecutter, Zope"
+    "description": "Manage Plone backend"
+    "property=og:description": "Manage Plone backend"
+    "property=og:title": "Manage Plone backend"
+    "keywords": "Plone 6, manage, backend, add-ons, packages, mxdev"
 ---
 
+(manage-plone-backend-label)=
 
-(manage-add-ons-packages-and-processes-label)=
+# Manage Plone backend
```

Comment:
I think in the future it might make sense to split this into "configure the Plone backend" and "manage add-ons" chapters. But that can happen later, when we add more material.

Reply:
Agreed.

I assume "add-ons" would retain the same content.

I assume "configure" would be a new page and include a port of https://5.docs.plone.org/adapt-and-extend/config/index.html. I'm loathe to start on this, however, until we have automated screenshots.

Reply:
Hmm, maybe. I was thinking of this as more the stuff you do outside of the Plone UI itself. It could be that too, but then we have to decide how to handle the Volto vs Classic divide for it.

Reply:
OK, that makes sense. I'd prefer to wait until we have automated screenshots before starting on any web UI stuff anyway.

Can you provide examples of what should go into "configure"? Dexterity content types? Other?